app/index.ejs

```diff
@@ -36,10 +35,49 @@
           },
         ],
       };
+      window.dataLayer = [];
     </script>
     <% if(htmlWebpackPlugin.options.prod) { %>
+    <!-- Google Tag Manager -->
+    <script>
+      (function(w, d, s, l, i) {
+        w[l] = w[l] || [];
+        w[l].push({ 'gtm.start': new Date().getTime(), event: 'gtm.js' });
+        var f = d.getElementsByTagName(s)[0],
+          j = d.createElement(s),
+          dl = l != 'dataLayer' ? '&l=' + l : '';
+        j.async = true;
+        j.src =
+          'https://www.googletagmanager.com/gtm.js?id=' +
+          i +
+          dl +
+          '&gtm_auth=VMsIb-u97IZIyagKFgXe1w&gtm_preview=env-1&gtm_cookies_win=x';
```

Comment:
I don't think we want `gtm_preview` in prod?

Reply:
I got that code from the Live environment at https://tagmanager.google.com/?authuser=1#/admin/accounts/1597949959/containers/35270741/environments?containerDraftId=2 but I'll use the one from the link on the main dashboard page.